conda_env/exceptions.py

```diff
@@ -2,6 +2,11 @@ class CondaEnvException(Exception):
     pass
 
 
+class NoBinstar(CondaEnvException):
+    def __init__(self):
+        msg = 'The binstar must be installed to perform this action'
```

Comment:
FWIW, I think it's mean to be "The binstar package" or "The binstar client". :-)